.github/workflows/workflow.yaml

```
@@ -0,0 +1,11 @@
+name: Build Test
+on: [push]
+
+jobs:
+  build_job:
+    runs-on: ubuntu-latest
```

I'd prefer to specify the Ubuntu release explicitely, because the workflow documentation currently lists `ubuntu-20.04`, and `ubuntu-18.04`, but lets us know that `ubuntu-latest` is `ubuntu-18.04` and not `ubuntu-20.04`.

```suggestion
    runs-on: ubuntu-20.04
```